app/serializers/bike_v2_serializer.rb

```diff
@@ -42,32 +41,43 @@ def title
   end
 
   def date_stolen
-    object.current_stolen_record && object.current_stolen_record.date_stolen.to_i
+    object.current_stolen_record&.date_stolen&.to_i
   end
 
   def thumb
-    if object.public_images.present?
-      object.public_images.first.image_url(:small)
-    elsif object.stock_photo_url.present?
+    image = object.public_images&.first&.image_url(:small)
+    return image if image.present?
+
+    if object.stock_photo_url.present?
       small = object.stock_photo_url.split("/")
       ext = "/small_" + small.pop
       small.join("/") + ext
     end
   end
 
   def large_img
-    if object.public_images.present?
-      object.public_images.first.image_url(:large)
-    elsif object.stock_photo_url.present?
+    object.public_images&.first&.image_url(:large).presence ||
       object.stock_photo_url
-    end
+  end
+
+  def url
+    "#{ENV["BASE_URL"]}/bikes/#{object.id}"
   end
 
   def is_stock_img
-    object.public_images.present? ? false : object.stock_photo_url.present?
+    object.public_images.blank? && object.stock_photo_url.present?
   end
 
   def stolen_location
-    object.current_stolen_record && object.current_stolen_record.address_short
+    stolen_record = object.current_stolen_record
```

Comment:
Why this change? What is wrong with the `address_short` method? If you're just adding the country here, why not add it to the address_short method?

Reply:
acfadad2